configs/etc/kubehound-reference.yaml

```diff
@@ -124,18 +121,26 @@ builder:
 
     #  # Cluster impact batch size for edge inserts
     # batch_size_cluster_impact: 1
-  
+
 # Ingestor configuration (for KHaaS)
 # ingestor:
 #   blob:
 #     # (i.e.: s3://<your-bucket>)
 #     bucket: ""
 #     # (i.e.: us-east-1)
-#     region: "" 
+#     region: ""
 #   temp_dir: "/tmp/kubehound"
 #   archive_name: "archive.tar.gz"
 #   max_archive_size: 2147483648 # 2GB
 #   # GRPC endpoint for the ingestor
-#   api: 
+#   api:
 #     endpoint: "127.0.0.1:9000"
 #     insecure: true
+
+#
+# Dynamic info (optionnal - auto injected by KubeHound)
+#
+dynamic:
+  #
+  #   # Target cluster name
+  #   cluster: <cluster name>
```

Comment:
Why commenting out the `cluster` but not the `dynamic` keys?
`dynamic` is optional and could be commented out like `ingestor` is above no?